### PR TITLE
Added text to the Array.map page

### PR DESCRIPTION
**Pull request by perborgen**
I'm also trying out embedding. It doesn't work in GitHub's markdown, but I read that it was suppose to work with embeds on the actual site.

### CODE REVIEW

File: src/pages/javascript/standard-objects/array/array-prototype-map/index.md [hunk 1]
```diff
@@ -3,11 +3,18 @@ title: Array.prototype.map
 ---
 ## Array.prototype.map
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/standard-objects/array/array-prototype-map/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The `.map()` method loops through the given array and executes a provided function on each of the element. It returns a new array which contains the the results of the function calls on each the elements.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+    var arr = [1, 2, 3, 4];
```
Comment (Bouncey) [inline src/pages/javascript/standard-objects/array/array-prototype-map/index.md]: please use ` ```js` here to format the code block and add syntax highlighting

File: src/pages/javascript/standard-objects/array/array-prototype-map/index.md [hunk 2]
```diff
@@ -3,11 +3,18 @@ title: Array.prototype.map
...
+    var newArray = arr.map(function(element) { return element * 2});
+    console.log(newArray); // [2, 4, 6, 8]
+
```
Comment (Bouncey) [inline src/pages/javascript/standard-objects/array/array-prototype-map/index.md]: you can close a code block with three back ticks.

File: src/pages/javascript/standard-objects/array/array-prototype-map/index.md [hunk 3]
```diff
@@ -3,11 +3,18 @@ title: Array.prototype.map
...
+
+Here is an interactive Scrimba screencast which explains `Array.prototype.map()`:
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+<div style="position: relative; padding-bottom: 56.25%;"><iframe allowfullscreen="true" src="https://scrimba.com/cast/c2Lg3hB.embed" style="border: 0px; position: absolute; width: 100%; height: 100%;"></iframe></div>
+
+<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 #### More Information:
```
Comment (Bouncey) [inline src/pages/javascript/standard-objects/array/array-prototype-map/index.md]: These last two lines can be removed

File: src/pages/javascript/standard-objects/array/array-prototype-map/index.md [hunk 4]
```diff
@@ -3,11 +3,18 @@ title: Array.prototype.map
 ---
 ## Array.prototype.map
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/standard-objects/array/array-prototype-map/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The `.map()` method loops through the given array and executes a provided function on each of the element. It returns a new array which contains the the results of the function calls on each the elements.
```
Comment (Bouncey) [inline src/pages/javascript/standard-objects/array/array-prototype-map/index.md]: >The `.map()` method loops through the given array and executes ~a~ provided function on each ~of the~ element. It returns a new array which contains ~the~ the results of the function call~s~ on each ~the~ element~s~.

The `.map()` method loops through the given array and executes the provided function on each element. It returns a new array which contains the results of the function call on each element.

File: src/pages/javascript/standard-objects/array/array-prototype-map/index.md [hunk 5]
```diff
@@ -3,11 +3,18 @@ title: Array.prototype.map
...
+    var newArray = arr.map(function(element) { return element * 2});
+    console.log(newArray); // [2, 4, 6, 8]
+
+
+Here is an interactive Scrimba screencast which explains `Array.prototype.map()`:
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+<div style="position: relative; padding-bottom: 56.25%;"><iframe allowfullscreen="true" src="https://scrimba.com/cast/c2Lg3hB.embed" style="border: 0px; position: absolute; width: 100%; height: 100%;"></iframe></div>
+
+<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
```
Comment (Bouncey) [inline src/pages/javascript/standard-objects/array/array-prototype-map/index.md]: This can be removed

**Comment by perborgen** [comment]
Hey @Bouncey ! Thanks a lot for the feedback. Just updated it. Let me know if there's anything else you want me to do.

**Comment by Bouncey** [comment]
Thanks @perborgen

**Pull request merged**